docs/cli/graphql-transformer/directives.md

```diff
@@ -476,60 +476,66 @@ type Post
 }
 ```
 
-Owner authorization specifies that a user can access an object. To
-do so, each object has an *ownerField* (by default "owner") that stores ownership information
-and is verified in various ways during resolver execution.
+Owner authorization specifies that a user can access an object. To do so, each object will get an `ownerField` field (by default "owner", will be added to the object if not specified) that stores ownership information and is verified in various ways during resolver execution.
 
-You can use the *operations* argument to specify which operations are augmented as follows:
+You can use the `operations` argument to specify which operations are enabled as follows:
 
 - **read**: If the record's owner is not the same as the logged in user (via `$ctx.identity.username`), throw `$util.unauthorized()` in any resolver that returns an object of this type.
-- **create**: Inject the logged in user's `$ctx.identity.username` as the *ownerField* automatically.
-- **update**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
-- **delete**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
+- **create**: Inject the logged in user's `$ctx.identity.username` as the `ownerField` automatically.
+- **update**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
+- **delete**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
 
-**Note**: When specifying operations as a part of the @auth rule, the operations not included in the list are not protected by default. For example, let's say you have the following schema:
+**Note**: When specifying operations as a part of the `@auth` rule, the operations not included in the list are not protected by default.
```

Comment:
This is an important note. Should we use `amplify-callout`?

Reply:
Good point, updated.